### PR TITLE
feat(categorize): categories per accept values up to 6-PD-1983

### PR DESCRIPTION
**Pull request by arimieandreea**

### CODE REVIEW

**Comment by arimieandreea** [comment]
The changes from categories.jsx are done to avoid the warning regarding the unique key in items in case of adding divs

**Review by PatriciaRomaniuc — APPROVED** [review]

File: packages/categorize/configure/src/design/index.jsx [hunk 1]
```diff
@@ -239,8 +240,8 @@ export class Design extends React.Component {
                   categoriesPerRow: categoriesPerRow.settings && numberField(categoriesPerRow.label,
                     {
                           label: categoriesPerRow.label,
-                          min: 1,
-                          max: 4,
+                          min: lowerMaximumValueCategoriesPerRow,
```
Comment (andreeimiron) [inline packages/categorize/configure/src/design/index.jsx:243]: the name is too long for this one. I think you can rename to `minCategoriesPerChoice` if I understand correctly what it refers to

**Review by andreeimiron — CHANGES_REQUESTED** [review]
You need to add the new config property in the documentation

File: packages/pie-models/src/pie/categorize/index.ts [hunk 2]
```diff
@@ -201,6 +201,11 @@ export interface CategorizeConfigure
    */
   maxCategories?: number;
 
+  /**
+   *  Lower maximum value for categories per row
```
Comment (PatriciaRomaniuc) [inline packages/pie-models/src/pie/categorize/index.ts:205]: Don't you mean minimum value? If that is what it represents

**Review by andreeimiron — APPROVED** [review]

**Pull request merged**